packages/protocol/scripts/truffle/make-release.ts

```diff
@@ -233,15 +238,8 @@ module.exports = async (callback: (error?: any) => number) => {
               )
               proposal.push({
                 contract: `${contractName}Proxy`,
-                function: '_setAndInitializeImplementation',
-                args: [contract.address, callData],
-                value: '0',
-              })
-            } else {
-              proposal.push({
-                contract: `${contractName}Proxy`,
-                function: '_setImplementation',
-                args: [contract.address],
+                function: 'initialize',
+                args: [callData],
```

Comment:
Remind me, does this work? Since we actually call the initialize function, not via a delegate call?

Reply:
it will resolve in the fallback function to a delegate call no?

Reply:
Yeah I think you are right